lua/apg/modules/ghosting.lua

```diff
@@ -316,28 +355,29 @@ local function checkDoor(ply, ent)
 				v:SetVelocity(push)
 			end
 		end
-		
+
 		timer.Simple(1, function()
 			if IsValid(ply) and IsValid(ent) then
 				ent.APG_Ghosted = false
 				ent:oldFadeDeactivate()
 				ent:SetCollisionGroup( COLLISION_GROUP_INTERACTIVE )
 
 				if IsValid(isTrap) then
-					APG.userNotification( "Unable to unstuck objects from fading door!", ply, 1 )
+					APG.log("CantUnstuck " .. ply:GetName() .. " from " .. ent)
+					APG.notification( "Unable to unstuck objects from fading door!", ply, 1 )
 					APG.entGhost(ent)
 				end
 			end
 		end)
 	end
 end
 
-APG.hookRegister(mod, "APG.FadingDoorToggle", "APG_FadingDoor", function(ent, isFading)
+APG.hookAdd(mod, "APG.FadingDoorToggle", "APG_FadingDoor", function(ent, isFading)
 	if APG.isBadEnt(ent) and APG.cfg["fadingDoorGhosting"].value then
 		local ply = APG.getOwner( ent )
 
-		if (IsValid(ply) and not isFading) then
-			timer.Simple(0.001, function() 
+		if (ply:IsPlayer() and not isFading) then
+			timer.Simple(0.001, function()

```

Comment:
Could you include a comment describing why this number was used here?

Reply:
While I'm not completely sure, I'm pretty sure it is used here because if you make a timer small enough it will call multiple times a second.

Reply:
Oh, must have misread, you didn't add this number. Disregard

Reply:
It's just to delay stuff from happening every so slightly, since this is basically a hack we need to wait for other things to happen for it to work.